src/sections/Hero/style.scss

```diff
@@ -25,22 +29,32 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  height: 16.25rem;
-  width: 16.25rem;
-
-  & + & {
-    margin-top: $grid-gutter-width;
+  height: 12rem;
+  width: 12rem;
 
-    @include media-breakpoint-up(xl) {
-      margin-top: 0;
-    }
+  @include media-breakpoint-up(md) {
+    height: 18vw;
```

Comment:
**Question:**`height: 18 viewport width`. Shouldn't this be related to the `viewport height`, it's strange to use a width specific unit in a height property.

Reply:
Yet totally agree with you, I wanted to use the same approach as the fonts for this particular items in order to keep the relation between this and the title. 

In that sense, as they are circles we need height and width to be equal, we get the same value we use for the width into height.